### PR TITLE
A little fixes.

### PR DESCRIPTION
**Pull request by nekonomicon**

### CODE REVIEW

**Comment by a1batross** [comment]
I think you can use #pragma once before the include guards, which may result a faster preprocessing. And it works almost everywhere.

**Comment by FreeSlave** [comment]
You reorganized layour of some variables. Will not it break saves?

File: dlls/effects.cpp [hunk 1]
```diff
@@ -379,17 +379,17 @@ class CLightning : public CBeam
 
 	void BeamUpdateVars( void );
 
+	string_t	m_iszStartEntity;
+	string_t	m_iszEndEntity;
+	string_t	m_iszSpriteName;
```
Comment (a1batross) [inline dlls/effects.cpp]: Don't break the offsets, Hamsandwich from AMXX relies on it.

File: dlls/func_break.h [hunk 2]
```diff
@@ -82,9 +82,9 @@ class CBreakable : public CBaseDelay
 
 	Materials m_Material;
 	Explosions m_Explosion;
+	string_t m_iszGibModel;
+	string_t m_iszSpawnObject;
```
Comment (a1batross) [inline dlls/func_break.h]: Same here

File: dlls/func_tank.cpp [hunk 3]
```diff
@@ -97,6 +97,9 @@ class CFuncTank : public CBaseEntity
 
 protected:
 	CBasePlayer* m_pController;
+	string_t	m_iszSpriteSmoke;
+	string_t	m_iszSpriteFlash;
+	string_t	m_iszMaster;	// Master entity (game_team_master or multisource)
```
Comment (a1batross) [inline dlls/func_tank.cpp]: And here.

File: dlls/lights.cpp [hunk 4]
```diff
@@ -37,8 +37,8 @@ class CLight : public CPointEntity
 	static TYPEDESCRIPTION m_SaveData[];
 
 private:
+	string_t m_iszPattern;
 	int m_iStyle;
-	int m_iszPattern;
```
Comment (a1batross) [inline dlls/lights.cpp]: And here

**Comment by a1batross** [comment]
Well. I will not comment every layout reorganization, just don't move them.

**Comment by nekonomicon** [comment]
Ok.

**Pull request merged**